### PR TITLE
add QUIC

### PR DESCRIPTION
**Pull request by marten-seemann**
I added QUIC at Code 81, which is the ASCII code point of "Q".

### CODE REVIEW

File: protocols.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ const (
 	P_UDP   = 17
 	P_DCCP  = 33
 	P_IP6   = 41
+	P_QUIC  = 81
```
Comment (whyrusleeping) [inline protocols.go]: 81 SGTM, please also PR it into the table here: https://github.com/multiformats/multicodec/blob/master/table.csv

**Comment by whyrusleeping** [comment]
LGTM

File: protocols.go (hunk 1 above)
Comment (marten-seemann) [inline protocols.go]: Doesn't this belong into https://github.com/multiformats/multiaddr?

Comment (Kubuxu) [inline protocols.go]: Multicodec unified table, so really both.

Comment (marten-seemann) [inline protocols.go]: Done, I sent two more PRs.

**Pull request merged**